jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/PluginConfigurationProcessor.java

```diff
@@ -230,4 +259,47 @@ MavenSettingsServerCredentials getMavenSettingsServerCredentials() {
   boolean isBaseImageCredentialPresent() {
     return isBaseImageCredentialPresent;
   }
+
+  /**
+   * Compute the container entrypoint, in this order :
+   *
+   * <ol>
+   *   <li>the user specified one, if set
+   *   <li>for a war project, the jetty default one
+   *   <li>for a jar project, by resolving the main class
+   * </ol>
+   *
+   * @param logger the logger used to display messages.
+   * @param jibPluginConfiguration the {@link JibPluginConfiguration} providing the configuration
+   *     data
+   * @param projectProperties used for providing additional information
+   * @return the entrypoint
+   * @throws MojoExecutionException if the http timeout system property is misconfigured
+   */
+  static List<String> computeEntrypoint(
+      Log logger,
+      JibPluginConfiguration jibPluginConfiguration,
+      MavenProjectProperties projectProperties)
+      throws MojoExecutionException {
+
+    List<String> entrypoint = jibPluginConfiguration.getEntrypoint();
```

Comment:
See the changes [here](https://github.com/GoogleContainerTools/jib/pull/1072/files#diff-48ecbd916f6fe40435ecf7339c54cee6R179) which I think is more readable.

Reply:
Done